README.md

```diff
@@ -0,0 +1,150 @@
+# vllm-orchestrator-gateway
+
+This gateway service enables controlled interaction with the FMS Guardrails Orchestrator by enforcing stricter access to its exposed endpoints. It provides a mechanism of configuring fixed detector pipelines, and then provides a unique /v1/chat/completions endpoint per configured detector pipeline. This allows for drop-in replacement of an unguardrailed chat completions model with a guardrailed one.
+
+### Getting started
+To see the entire stack of the vllm-orchestrator-gateway there are a few services that need to be spun up. Some of these can be swapped for other services that use follow certain api's, such as the detectors and generation models.
+
+- [FMS Guardrails Orchestrator](https://github.com/foundation-model-stack/)
+- [Guardrails Regex Detector](https://github.com/trustyai/guardrails-regex-detector)
+- [VLLM using Qwen/Qwen2.5-1.5B-Instruct](https://docs.vllm.ai/en/latest/getting_started/quickstart.html#openai-compatible-server)
+
+
+### Sample config
+The config has 3 main fields, `orchestrator`, `detectors` and `routes`.
+
+`orchestrator` is where the `orchestrator` service lives.
+
+`detectors` are detectors services that have been defined in the `fms-guardrails-orchestrator` config file. You can specify what detector belongs to input and/or output.
+
+`routes` are the dynamically exposed routes used to enforce detectors onto endpoints such as the `pii` endpoint that registers our `regex-language` detector. You can also specify no detectors such as the `passthrough` route down below.
+
+```yaml
+orchestrator:
+  host: localhost
+  port: 8085
+detectors:
+  - name: regex-language
+    input: false
+    output: true
+    detector_params:
+      regex:
+        - email
+        - ssn
+routes:
+  - name: pii
+    detectors:
+      - regex-language
+    fallback_message: "I'm sorry, I'm afraid I can't do that."
```

Comment:
We should example the `fallback_message` argument

Reply:
I have showcased the `fallback_message` in the `sample response with detection` section below or do you mean provide an explanation in the above config description?

Reply:
@RobGeada updated with 
`fallback_message` in the `routes` field is used as a response from the gateway when a detection is found either in the input or output.